gradle/license-header.txt

```diff
@@ -5,7 +5,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *      https://www.apache.org/licenses/LICENSE-2.0
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why not HTTPS?

Reply:
Because otherwise it's 200+ file changes and I wanna decrease the noise as much as possible. I'll follow up